### PR TITLE
EASYOPAC-1224 - Various EO bugs.

### PR DESCRIPTION
**Pull request by NicolaeGrigoriu**
#### Link to issue

https://inlead.atlassian.net/browse/EASYOPAC-1224

#### Description

- Remove duplicated ding campaign plus widget feature from the search results page
- Update filter list of default IPE items
- Consider price fields which have value 0 as free
- Added the easyddb_frontend_layout for the sections builder

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Comment by guddo** [comment]
Actually it seems like ding_campaign_plus is also duplicated on `ting_object` and `ting_collection` panels..

File: modules/ding_ipe_filter/ding_ipe_filter.install [hunk 1]
```diff
@@ -140,3 +140,11 @@ function ding_ipe_filter_update_7106(&$sandbox) {
   variable_set('ding_ipe_filter_panes_selected', $new_selections);
 }
 
+/**
```
Comment (guddo) [inline modules/ding_ipe_filter/ding_ipe_filter.install]: Can't this be moved to easyddb_frontend_layout?

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1fd1975ffd845e1411ba944a377ded10f704cc41
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

File: modules/ding_ipe_filter/ding_ipe_filter.install (hunk 1 above)
Comment (NicolaeGrigoriu) [reply modules/ding_ipe_filter/ding_ipe_filter.install]: https://github.com/easySuite/easyddb_frontend_layout/pull/22

**Pull request merged**